source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+=======================
+Halloween Treasure Hunt
+=======================
+
+.. contents:: Table of Contents
+    :local:
+
+Introduction
+============
+
+Welcome back to the **Denizen-for-Bukkit Docs**. This is our first example project, so we'll start with the basics. 
+This guide will cover the following topics:
+
+* Denizen objects and general syntax.
+* Browsing through the documentation.
+* Reloading scripts and debug modes.
+* World script containers, events and context tags.
+* Common script commands and arguments.
+* Constructing tags with simple bases and modifiers.
+* First contact with foreach loops.
+
+If you are not familiar with any of the concepts listed, we suggest revisiting the following tutorial documents:
+
+* :doc:`Section 1.3 - dScript Format </docs/getting-started/dscript-format>`
+* :doc:`Section 2.1 - An Introduction to Tags </docs/basics-of-scripting/an-introduction-to-tags>`
+* :doc:`Section 2.2 - The Basics of Denizen Commands </docs/basics-of-scripting/basics-of-denizen-commands>`
+* :doc:`Section 2.7 - Your First World Script </docs/basics-of-scripting/your-first-world-script>`
+* foreach loops here
+
+This will help you make a better use of this example project, and follow its contents successfully.
+
+.. note::
+    It is also recommended to have the `meta documentation <https://one.denizenscript.com/denizen/logs>`_ open in 
+    another tab while scripting this project, as we'll need to check the syntax of events, commands and tags very 
+    often.
+
+First Steps
+===========
+
+Now let's jump (or slowly crawl) into the scripting process. In our case, we want to make a **Halloween Treasure 
+Hunt** event in our *Hub world*. We plan on manually placing some hidden pumpkins ourselves, and reward players for 
+finding (and left clicking) them.
+
+First of all, we'll need to create a *new script file* (for example ``Halloween_Treasure_Hunt.yml``) and build a world 
+script container which, as we know, will listen to events happening within our server. We just have to give it a name, 
+like ``Halloween_Treasure_Hunt``, and set the ``type:`` key to ``world``. In addition to this, we'll also enable debug 
+by setting the ``debug:`` key to ``true``. This will make our script print in the console everything it does and help 
+us solve errors. It is not needed, so we can just disable it later on. We'll now add an ``events:`` subkey, which will 
+in the end hold the executable code we're going to write.
```

Comment:
All of this information should be covered in the sections specified from the list on line 24, so there's no need to be super-explicit about what you're doing here. You can say that you're creating a new script in presumably a new file, and that you're using a world script named `Halloween_Treasure_Hunt` (I would prefer the script name to be all-lowercase, since Denizen is case-insensitive with those, but this is fine I suppose). If you need to explicitly note the use of the `debug:` key (as in setting it to `false`), you can do so.